server/prisma/prisma.yml

```diff
@@ -1,21 +1,30 @@
-# Specifies the HTTP endpoint of your Prisma API (deployed to a Prisma Demo server).
-endpoint: https://us1.prisma.sh/public-mirrorlifter-291/react-fullstack/dev
+endpoint: https://us1.prisma.sh/felipe-valencia/hr-kraken/dev
 
 # Defines your models, each model is mapped to the database as a table.
-datamodel: datamodel.prisma
+datamodel: 
+  - ./model/candidate.graphql
+  - ./model/application.graphql
+  - ./model/result.graphql
+  - ./model/status.graphql
+  - ./model/opening.graphql
+  - ./model/application-step.graphql
+  - ./model/step.graphql
+  - ./model/template-step.graphql
 
 # Specifies the language and directory for the generated Prisma client.
 generate:
   - generator: javascript-client
     output: ../src/generated/prisma-client
+  - generator: graphql-schema
+    output: ../src/generated/schema.graphql
 
 seed:
   import: seed.graphql
 
 # Ensures Prisma client is re-generated after a datamodel change.
-hooks:
-  post-deploy:
-    - prisma generate
+# hooks:
```

Comment:
shouldn't this be uncommented? to generate again our prisma client automatically if we make a deploy